src/friendly-iframe-embed.js

```diff
@@ -511,29 +516,22 @@ export class FriendlyIframeEmbed {
    * @return {!Promise}
    */
   enterFullOverlayMode() {
-    const iframeBody = this.getBodyElement();
-    const fixedContainer = this.getFixedContainer();
-
     return this.runVsyncOnIframe_({
       measure: state => {
         const iframeRect = this.iframe./*OK*/getBoundingClientRect();
 
-        const winWidth = this.win./*OK*/innerWidth;
-        const winHeight = this.win./*OK*/innerHeight;
-
-        state.fixedContainerStyle = {
+        state.bodyStyle = {
+          'background': 'transparent', // TODO(alanorozco): do this early
```

Comment:
do you plan to talk about body transparency in a separate PR (in a detailed README)?